README.md

```diff
@@ -41,11 +41,11 @@ This is the most basic case.
            Gross Salary: £416.67
 
 Calculation rules:
- * Monthly Gross Salary: The monthly gross salary is the employee's annual gross salary divided by 12
+ * "Gross Salary" on a salary slip is equal to the employee's annual gross salary divided by 12.
```

Comment:
I think that's part of the interesting feature. Figuring out the language.
Exactly like an actual spec, the vocabulary can be fickle